### PR TITLE
Pcs 89 shared pipeline

### PR DESCRIPTION
**Pull request by ghost**
This is just the seeds of a project where we can place definitions of partial and full pipelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,15 @@ Reusable pipeline-as-code snippets. The use of these snippets depends on the def
  to this repository. At the current time, this is done in the digital-probation-tooling project which deploys Jenkins in Kubernetes. 
 
 # Usage
-The snippets can be used as follows. This shows the use of simplePipeline, via a library definition named "digital-probation-pipeline"
+The snippets can be used as follows. This shows the use of simplePipeline, via a library definition named "digital-probation-pipeline". A branch
+ name can be added as shown and could be drawn from environment variables.
```
Comment (robinsillem) [inline README.md]: What repo is this a branch of? What is its purpose?

File: vars/gradlePipeline.groovy [hunk 2]
```diff
@@ -0,0 +1,56 @@
+def call(Map pipelineParams) {
```
Comment (robinsillem) [inline vars/gradlePipeline.groovy]: Presumably this parameter is a placeholder for some later functionality?

**Review by robinsillem — APPROVED** [review]

File: vars/gradlePipeline.groovy (hunk 2 above)
Comment (ghost) [reply vars/gradlePipeline.groovy]: Yes you can pass parameters in. I am likely to use it again for the docker build. If not, I'll remove it then.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: I've added to the README

**Pull request merged**